test/browser/components.js

```diff
@@ -214,31 +214,31 @@ describe('Components', () => {
 
 		let good, bad;
 		let root = render(<GoodContainer ref={c=>good=c} />, scratch);
-		expect(scratch.innerText, 'new component with key present').to.equal('A\nB');
+		expect(scratch.textContent, 'new component with key present').to.equal('AB');
```

Comment:
Because `innerText` returns content as is (as it was entered in markup). `textContent` is more "strict" and behaves as specced (result depends on nodes' positions).